### PR TITLE
bootspeed: debugging printouts if systemd timeout is reached

### PR DESCRIPTION
**Pull request by paride**

### CODE REVIEW

**Comment by panlinux** [comment]
I would also suggest a process listing dump, like `ps fauxw`

**Comment by paride** [comment]
Thanks @panlinux

File: boot-speed/bootspeed.sh [hunk 1]
```diff
@@ -52,6 +52,8 @@ if timeout 20m sh -c "until systemd-analyze time; do sleep 20; done"; then
     systemd-analyze critical-chain > systemd-analyze_critical-chain
     systemd-analyze plot > systemd-analyze_plot.svg
 else
```
Comment (OddBloke) [inline boot-speed/bootspeed.sh]: The output of `systemd-analyze time` in the failure case includes "Hint: Use 'systemctl list-jobs' to see active jobs", is it worth including the output of `systemctl list-jobs` here? (`blame` may capture the same info, I'm not sure.)

Comment (paride) [reply boot-speed/bootspeed.sh]: Yeah I'm refactoring a bit that part of the script and added a `list-jobs`. Thanks!

**Pull request merged**